backend/seed_data/hints.yaml

```
@@ -49,7 +49,28 @@ hints:
+      - text: "Return true if every character in s is matched with a character in t."
+        order: 3
+
+  - module_type: "BONUS_CHALLENGE"
+    module_title: "Product of Array Except Self"
+    hints:
+      - text: "Use the multiple passes strategy, including traversing from the array from the right."
+        order: 1
+      - text: "Consider breaking down the products into smaller parts. Can you separate the numbers that come before the current index from those that come after it?"
+        order: 2
+      - text: "Track the running products of the numbers before and after the current index in the outcome array."
+        order: 3
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)